src/main/java/com/telefonica/iot/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -130,7 +130,7 @@ public void configure(Context context) {
         mysqlPassword = context.getString("mysql_password", "unknown");
```

Comment:
I've seen this "unknown" default password... I guess all the default passwrods are empty in the templates, right?